python/cugraph/structure/graph.py

```diff
@@ -565,945 +277,394 @@ def from_numpy_matrix(self, np_matrix):
-        index_col arguments wrap gdf_column objects that represent a graph
-        using the adjacency list format.
-        If value_col is None, an unweighted graph is created. If value_col is
-        not None, a weighted graph is created.
-        Undirected edges must be stored as directed edges in both directions.
-
+        Given a DataFrame containing external vertex ids in the identified
+        columns, or a Series containing external vertex ids, return a
+        Series with the internal vertex ids.
+        Note that this function does not guarantee order in single GPU mode,
+        and does not guarantee order or partitioning in multi-GPU mode.
         Parameters
         ----------
-        offset_col : cudf.Series
-            This cudf.Series wraps a gdf_column of size V + 1 (V: number of
-            vertices).
-            The gdf column contains the offsets for the vertices in this graph.
-            Offsets must be in the range [0, E] (E: number of edges).
-        index_col : cudf.Series
-            This cudf.Series wraps a gdf_column of size E (E: number of edges).
-            The gdf column contains the destination index for each edge.
-            Destination indices must be in the range [0, V) (V: number of
-            vertices).
-        value_col : cudf.Series, optional
-            This pointer can be ``None``.
-            If not, this cudf.Series wraps a gdf_column of size E (E: number of
-            edges).
-            The gdf column contains the weight value for each edge.
-            The expected type of the gdf_column element is floating point
-            number.
-
-        Examples
-        --------
-        >>> gdf = cudf.read_csv('datasets/karate.csv', delimiter=' ',
-        >>>                   dtype=['int32', 'int32', 'float32'], header=None)
-        >>> M = gdf.to_pandas()
-        >>> M = scipy.sparse.coo_matrix((M['2'],(M['0'],M['1'])))
-        >>> M = M.tocsr()
-        >>> offsets = cudf.Series(M.indptr)
-        >>> indices = cudf.Series(M.indices)
-        >>> G = cugraph.Graph()
-        >>> G.from_cudf_adjlist(offsets, indices, None)
-
+        df: cudf.DataFrame, cudf.Series, dask_cudf.DataFrame, dask_cudf.Series
+            A DataFrame containing external vertex identifiers that will be
+            converted into internal vertex identifiers.
+        column_name: (optional) string
+            Name of the column containing the external vertex ids
+        Returns
+        ---------
+        series : cudf.Series or dask_cudf.Series
+            The internal vertex identifiers
         """
-        if self.edgelist is not None or self.adjlist is not None:
-            raise Exception("Graph already has values")
-        self.adjlist = Graph.AdjList(offset_col, index_col, value_col)
-
-        if self.batch_enabled:
-            self._replicate_adjlist()
+        return self.renumber_map.to_internal_vertex_id(df, column_name)
```

Comment:
`self.renumber_map` no longer exists in the `graph` class.  Similar recommendation as above if I am not wrong